src/ui/ImageSpan.js

```diff
@@ -103,4 +103,15 @@ export default class ImageSpan extends Component {
       {width, height}
     );
   }
+
+  _getSize() {
+    let {width, height} = this.state;
+    if (!isNaN(width)) {
+      width = `${width}px`;
+    }
+    if (!isNaN(height)) {
+      height = `${height}px`;
+    }
+    return {width, height};
```

Comment:
Hmm, I'm not clear on why we need this logic; What's the case where one of these would be `NaN`?

If somehow a non-number (or invalid number) could sneak into the state for width/heigh, I'm assuming it's up on line 35. Should we do the validation there instead?

Reply:
<img/> support both number and string for width and height, so we need this logic to handle both case:
<img width="100" height="100"/> 
<img width="100px" height="100px"/>

And I agree it is better to do validation on line 35. I was trying to limit the PR size, I thought change on line 66 is safer. 
Do you want me move it to line 35?